toptal/picasso
# chore: update lerna to v5

[FX-2859](https://toptal-core.atlassian.net/browse/FX-2859)

### Description

Updating lerna to v5. I didn't find any relevant performance boosts for our use cases when running `build:package` (the only script still using lerna on Picasso). In fact when using `useNx: true` it was twice as slower, even thought it is expected to be a lot faster.

### How to test

- CI should be green
- You can run `yarn build:package` on the local branch, as it uses lerna

### Screenshots

| Version                      | Benchmark |
| --------------------------------------- | --- |
| Lerna v4 | ![image](https://user-images.githubusercontent.com/5845160/176086702-fa208a69-db1c-4019-bd31-748f30e34dcb.png) |
| Lerna v5 | ![image](https://user-images.githubusercontent.com/5845160/176086788-4aaacde1-0a79-4547-ad7e-e1c42b578a5a.png) |
| Lerna v5 with useNx | ![image](https://user-images.githubusercontent.com/5845160/176086839-a53018d6-7708-4646-b8a4-85a6b8237423.png) |

### Development checks

- Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
-  Annotate all `props` in component with documentation
- Create `examples` for component
- Ensure that deployed demo has expected results and good examples
- Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- Covered with tests

**Breaking change**

- codemod is created and showcased in the changeset
- test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>
